client/src/components/AddNewItem/AddNewItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Button, Grid, MenuItem, Select, Typography } from '@material-ui/core';
+import Input from '@material-ui/core/Input';
+import * as React from 'react';
+import useStyles from './useStyles';
+
+export default function AddNewItem(): JSX.Element {
+  const classes = useStyles();
+  const [list, setList] = React.useState('');
+  const [lists, setLists] = React.useState(['Clothes', 'Furniture', 'Luxury']);
+
+  const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setList(event.target.value as string);
+  };
+
+  return (
+    <Grid container justify="center" alignItems="center" direction="column" className={classes.root}>
+      <Typography variant="h5">
+        <Box fontWeight={700}>Add new item:</Box>
+      </Typography>
+      <form className={classes.form}>
+        <Input disableUnderline={true} placeholder="Paste your link here" className={classes.input}></Input>
+
+        <Select
+          value={list as string}
+          onChange={handleChange}
+          disableUnderline
+          displayEmpty
+          className={`${classes.select} ${list === '' ? classes.selectInitialState : ''}`}
+          inputProps={{ 'aria-label': 'Without label' }}
+        >
+          <MenuItem value="" disabled>
+            Select list
+          </MenuItem>
+          {lists.map((item: string, index: number) => (
+            <MenuItem key={index} value={item}>
+              {item}
+            </MenuItem>
+          ))}
+        </Select>
+        <Button variant="contained" color="primary" style={{ margin: '.5rem', width: '7rem' }}>
```

Comment:
You should avoid using inline styles, these can be moved into a class. Ditto throughout

Reply:
I only did this one inline because it was a very specific and small adjustment and was not gonna be used anywhere else in the code. But now I moved it inside its own class